### DISCUSSION TITLE
Malware/Phishing query log not working correctly

### DISCUSSION THREAD

**Discussion by Xevy17** [AdguardTeam/AdGuardHome#6849] [Q&A]
Antes de nada me disculpo si no escribo el mensaje en el sitio correcto. En la página inicial de AdGuard Home me aparece que se ha bloqueado X número de Malware/phishing, al hacer clic para ver el registro de esas consultas se queda un rato cargando y nunca aparece nada en el registro, alguna solución?

Gracias de antemano y disculpas de nuevo.

**Comment by ghost** [comment]
Have you got a screen recording that shows what you're referring to?

**Reply by Xevy17** [reply]
Aquí la tienes: 

https://github.com/AdguardTeam/AdGuardHome/assets/164785241/0c8881f8-a2b8-4b80-b793-c99a025ca869

**Reply by ainar-g** [reply]
This looks like a timeout error.  It is likely that there are a lot of queries in the query log, and AdGuard Home is unable to find enough of them in time.  You can [enable verbose logging](https://adguard-dns.io/kb/adguard-home/faq/#verboselog) to see if it is so.  This should also make it easier to find errors, if there are any.

**Comment by ainar-g** [comment]
Statistics and query logs are stored separately, so it could be that your statistics retention period is larger than the (double of) query log rotation period.  In that case, the statistics will retain information about queries that are no longer in the query log.

**Reply by Xevy17** [reply]
Gracias por responder, tengo configuradas las estadísticas para que se guarden durante 30 días, mi instalación tiene unos 15días así que deberían estar guardadas las consultas.
